the-badass-todo-list-part-1/main.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
   // Kick off
+  // Quiz#3
 document.addEventListener('DOMContentLoaded', function(event) {
   TodoListManager.init();
```

Comment:
Yes...that's one issue I always wonder why Gaia people named things like this so thoughtlessly. However, I need to remind you that `component` is actually another pattern in the book by GoF, so it still may cause some confusions.